daml/Marketplace/Custody.daml

```diff
@@ -87,18 +87,18 @@ template Service
         with
           openAccountRequestCid : ContractId OpenAccountRequest
         do
-          OpenAccountRequest{..} <- fetchAndArchive openAccountRequestCid
-          let
-            account = Account with provider = provider; owner = customer; id = accountId
+          OpenAccountRequest{accountId; observers; ctrls} <- fetchAndArchive openAccountRequestCid
+
+          let account = Account with provider = provider; owner = customer; id = accountId
```

Comment:
Shorthand for `X with y = y` is `X with y` :wink:

Reply:
I've updated this, thanks. - I didn't write this statement originally, I just brought up the `account = ...` line as there's only one variable being set so there was no reason for it to be on a separate line.